lms/djangoapps/discussion/static/discussion/templates/discussion-home.underscore

```diff
@@ -1,62 +1,85 @@
 <div class="discussion-article view-discussion-home">
-  <section class="home-header">
-    <span class="label"><%- gettext("Discussion Home") %></span>
-    <% if (window.courseName) { %>
-      <h2 class="home-title"><%- window.courseName %></h2>
-    <% } %>
-  </section>
-
   <% if (window.ENABLE_DISCUSSION_HOME_PANEL) { %>
-    <span class="label label-settings">
-      <%- interpolate(
+
+      <% if (document.getElementsByTagName("body")[0].className.indexOf("new-theme") == -1) { %>
```

Comment:
@umarmughal824 I would not recommend this approach. You should find some appropriate way to fix the issue, something like passing a variable from backend to decide which ui to enable.